backend/company_api.py

```
@@ -0,0 +1,776 @@
+        steps=progress.steps,
+        errors=progress.errors,
+        started_at=progress.started_at,
+        completed_at=progress.completed_at,
+        message="Onboarding started successfully"
+    )
+
+
+@router.post("/{company_id}/onboarding/pause", response_model=OnboardingProgressResponse)
+async def pause_onboarding(
+    company_id: str = Path(..., description="Company ID"),
+    user: dict = Depends(_get_current_user_thunk)
```

Ruff (0.15.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)